### PR TITLE
Changes for make tools

### PR DESCRIPTION
**Pull request by rsajwani**
Signed-off-by: Rameez Sajwani <rameezwazirali@hotmail.com>

## Description

1. 'Make Tools' on Mac M1 is failing
2. Following unit tests were failing on Mac M1
- go test vitess.io/vitess/go/vt/topo/k8stopo
- go test vitess.io/vitess/go/vt/zkctl
- go test vitess.io/vitess/go/vt/topo/zk2topo
3. Updated versions of zk, etcd, consul for distribution.
- check out my PR https://github.com/vitessio/vitess-resources/pull/8
- New vitess resource release https://github.com/vitessio/vitess-resources/releases/tag/v2.0

## Related Issue(s)
https://github.com/vitessio/vitess/issues/10093

## Checklist

-   [ ] "Backport me!" label has been added if this change should be backported. NO
-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required

## Testing

1. Ran 'Make Tools' on Mac M1 
2. Ran 'Make Tools' on Linux
3. Ran 'Make Tools' on Mac but non-M1 machine.

### CODE REVIEW

**Review by deepthi — COMMENTED** [review]
I expected `ensure_bootstrap_updated` check to fail because we are changing bootstrap.sh.

Looks like the path filtering is incorrect and we need to fix it by adding `bootstrap.sh` to the list here:
https://github.com/vitessio/vitess/blob/main/.github/workflows/ensure_bootstrap_updated.yml#L18

We also need to build and push new bootstrap images (and update the bootstrap image version in the Makefile).

**Review by GuptaManan100 — APPROVED** [review]
This looks good to me, but the proto check is failing.

**Comment by vmg** [comment]
Changes look fine, but since you forgot to bump the bootstrap version in `test.go`, it won't be enough to simply re-build and push the docker images now. The caches in CI won't pick up the changes. You're going to need to bump the version _again_ everywhere, and then push those images. That will make the build pass.

File: .github/workflows/ensure_bootstrap_updated.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
             - 'docker/**'
             - 'test.go'
             - 'Makefile'
+            - 'bootstrap.sh'
```
Comment (frouioui) [inline .github/workflows/ensure_bootstrap_updated.yml:22]: ❤️

**Review by frouioui — APPROVED** [review]
looks good to me!

File: test.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ For example:
 // Flags
 var (
 	flavor           = flag.String("flavor", "mysql57", "comma-separated bootstrap flavor(s) to run against (when using Docker mode). Available flavors: all,"+flavors)
-	bootstrapVersion = flag.String("bootstrap-version", "6", "the version identifier to use for the docker images")
+	bootstrapVersion = flag.String("bootstrap-version", "7", "the version identifier to use for the docker images")
```
Comment (frouioui) [inline test.go:77]: @vmg, the bootstrap version is bumped in `test.go`

Comment (vmg) [reply test.go:77]: Yes, but that happened after @deepthi pushed the docker images, so they need to be pushed _again_ (this is my understanding from reading the Slack logs).

Comment (deepthi) [reply test.go:77]: Makes sense. we'll do this today.

Comment (rsajwani) [reply test.go]: I have push a commit with all the files updated to 8 .. @deepthi

Comment (deepthi) [reply test.go]: Pushed new bootstrap images.

**Pull request merged**